src/components/Projects/Welcome/Welcome.js

```diff
@@ -8,8 +8,8 @@ const Layout = styled.div`
   grid-gap: 1em;
 `;
 
-const P = styled.div`
-    color: C4C4C4;
+const P = styled.p`
```

Comment:
This should be a styled Typography from the Elements folder. 

const StyledTypography = styled(Typography)`color: #8A8C98;`

Reply:
You would have to import Typography from common/Elements